pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -454,13 +479,12 @@ func (c *Cluster) startMon(m *monConfig, hostname string) error {
 	return nil
 }
 
-func waitForQuorumWithMons(context *clusterd.Context, clusterName string, mons []string) error {
+func waitForQuorumWithMons(context *clusterd.Context, clusterName string, mons []string, sleepTime int, requireAllInQuorum bool) error {
 	logger.Infof("waiting for mon quorum with %v", mons)
 
 	// wait for monitors to establish quorum
```

Comment:
Something I've wondered fort a while: why do we use a retry counter here instead of a timeout? It seems to me like that's a little more straightforward to understand how long the overall wait time is.

Reply:
I was thinking that the loop would largely be the same, but its timing would be governed by a timeout rather than a counter so that both checks would still be getting done. It's not something that I think needs changed immediately either way.